ee/modules/381-l2-load-balancer/hooks/alert_module_deprecated_l2lb.go

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2024 Flant JSC
+Licensed under the Deckhouse Platform Enterprise Edition (EE) license. See https://github.com/deckhouse/deckhouse/blob/main/ee/LICENSE
+*/
+
+package hooks
+
+import (
+	"github.com/flant/addon-operator/pkg/module_manager/go_hook"
+	"github.com/flant/addon-operator/pkg/module_manager/go_hook/metrics"
+	"github.com/flant/addon-operator/sdk"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+var _ = sdk.RegisterFunc(&go_hook.HookConfig{
+	OnBeforeAll: &go_hook.OrderedConfig{Order: 10},
+	Kubernetes: []go_hook.KubernetesConfig{
+		{
+			Name:       "l2loadbalancers",
+			ApiVersion: "network.deckhouse.io/v1alpha1",
+			Kind:       "L2LoadBalancer",
+			FilterFunc: applyLoadBalancerObjFilter,
+		},
+	},
+}, alertModuleDeprecatedL2lbExists)
+
+func applyLoadBalancerObjFilter(obj *unstructured.Unstructured) (go_hook.FilterResult, error) {
+	var l2loadbalancer L2LoadBalancer
+
+	err := sdk.FromUnstructured(obj, &l2loadbalancer)
+	if err != nil {
+		return nil, err
+	}
+	return true, nil
+}
+
+func alertModuleDeprecatedL2lbExists(input *go_hook.HookInput) error {
+	if len(input.Snapshots["l2loadbalancers"]) > 0 {
```

Comment:
use len(...) to specify in the alert message how many such objects are in the cluster

Reply:
One metric per two hooks is used, so applying len is not a good idea.